sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -110,7 +110,8 @@ class ThriftServerQueryTestSuite extends SQLQueryTestSuite with SharedThriftServ
     // VARIANT type
     "variant/named-function-arguments.sql",
     // SPARK-51516: Support the TIME data type by Thrift Server
-    "time.sql"
+    "time.sql",
+    "view-with-default-collation.sql"
```

Comment:
Can we put it near the existing collation golden files in this ignore list? e.g. line 107